Examples/Algorithms/Digitization/include/ActsExamples/Digitization/SmearingAlgorithm.hpp

```diff
@@ -72,39 +73,21 @@ class SmearingAlgorithm final : public BareAlgorithm {
     bool configured = false;
   };
 
-  /// Construct the digitization algorithm.
+  /// Construct the smearing algorithm.
   ///
   /// @param cfg is the algorithm configuration
   /// @param lvl is the logging level
   SmearingAlgorithm(Config cfg, Acts::Logging::Level lvl);
 
-  /// Build measurement from simulation hits at input
+  /// Build measurement from simulation hits at input.
   ///
   /// @param ctx is the algorithm context with event information
-  ///
   /// @return a process code indication success or failure
   ProcessCode execute(const AlgorithmContext& ctx) const final override;
 
  private:
   /// The configuration struct containing the smearers
   Config m_cfg;
-
```

Comment:
I saw you moved that to the source file, good call, we should do that wherever possible.

Reply:
This also allows us to  move most of the includes to the source file.